src/commands/go-test-legacy.yaml

```diff
@@ -0,0 +1,48 @@
+steps:
+  - run:
+      name: Configuring Go Environment
```

Comment:
Can we follow this pattern for names: https://github.com/giantswarm/architect-orb/pull/71/files#diff-9f10d1e31cf5e578c665e884ed6d3d02R3

I think this is more useful for the orb. I want to document that but I haven't found time yet.

Reply:
I was trying to follow the CircleCI style. They have job names like `Spin Up Environment` and `Persisting to Workspace`. Prefixing with the orb command name seems reasonable though.

Reply:
Yeah I know what you mean. We also have fmt entry about that. But I'd rather be practical here.

Reply:
Okay it's answered here https://github.com/giantswarm/architect-orb/pull/75#discussion_r384554291

Reply:
I've added the orb+command name prefix to step names now.